assignment.md

```diff
@@ -1,23 +1,31 @@
-What is the difference between push, pull, and fetch?
+# Important Git commands to understand 
 
-# Understanding git commands push, pull, and fetch
+To assist you in understanding Git concepts and related commands, Git command definitions are provided below.
```

Comment:
This section needed an introduction of the commands to be examined.   Added the introduction.